{{cookiecutter.project_name}}/server/main_app/templates/main_app/index.html

```diff
@@ -9,6 +9,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <link rel="stylesheet" href="{% static 'main_app/css/index.css' %}">
+
+     <link rel="icon" type="image/png" sizes="16x16" src=" {"/images/favicon-16x16.png"}">
```

Comment:
1. You are missing `{% static %}`
2. You are missing files that you are trying to include: you have submitted `favicon.png`. But you are including `favicon-16x16.png`
3. You have created `images/` folder in the wrong directory. You need to change it to  `static/images`. Because you can not include static files from templates directory